perf/perftool.py

```diff
@@ -39,7 +39,15 @@ def setUp(self):
 
         # Check for basic utilities
         smm = SoftwareManager()
-        for package in ['gcc', 'make', 'perf']:
+        detected_distro = distro.detect()
+        kernel_ver = platform.uname()[2]
+        if 'Ubuntu' in detected_distro.name:
+            deps = ['gcc', 'make', 'linux-tools-common',
```

Comment:
can gcc, make package is kind pre defined in deps as it is kind of common 

ex: 
```
deps=[''gcc', 'make'']
if 'Ubuntu' in detected_distro.name:
 deps.extend(['linux-tools-common',  'linux-tools-%s' % kernel_ver ])
else:
   deps.append('perf')
```